packages/app/src/cli/services/app/config/link.ts

```diff
@@ -51,7 +67,7 @@ export default async function link(options: LinkOptions, shouldRenderSuccess = t
         [`Make updates to ${configFileName} in your local project`],
         [
           'To upload your config, run',
-          {command: formatPackageManagerCommand(localApp.packageManager, 'shopify app config push')},
+          {command: formatPackageManagerCommand(localApp.packageManager, 'shopify app deploy')},
```

Comment:
Does it make sense to expose this messaging yet if this is just merging to `main`? Would a release be cut before we're ready to release?